.pre-commit-config.yaml

```diff
@@ -44,4 +44,4 @@ repos:
     hooks:
       - id: sops-encryption
         # Add files here if they contain the word 'secret' but should not be encrypted
-        exclude: secrets\.md
+        exclude: secrets\.md|helm-charts/support/templates/prometheus-ingres-auth/secret\.yaml
```

Comment:
Does this have to be a list of specific files, or can we make it "any file under helm-charts/**/templates/** that has secret in it's name"? Like in our git-ignore?

https://github.com/2i2c-org/infrastructure/blob/master/.gitignore#L16

Reply:
This must be a python regular expression i think, so i dont think so :/

Reply:
It's not critical, just a wondering :)